train.py

```diff
@@ -11,10 +11,10 @@
 random.seed(42)
 
 parser = argparse.ArgumentParser()
-parser.add_argument('network', type=str, choices=network.SUPPORTED_NETWORKS, help='Select the network to use for training')
```

Comment:
We should do this in a separate PR. Each PR should be self-contained.

Reply:
Oh! Okay! I will remove it here.